js/modules/questionFormValidation.js

```diff
@@ -0,0 +1,76 @@
+export default function questionFormValidation(){
+	
+	inputValidation();
+	checkboxQuestionsValidation();
+
+	function inputValidation(){
```

Comment:
function's name must reflect what this function does and there must be verb in this name.
for example
```js
	function validate(){}
```
or if this function must return boolean, it can be called like this
```js
 function isValid(){}
```

Reply:
Done